Automattic/wp-calypso
# Jetpack Logo Generator: add rough PoC for image generation

Add token and image generation request

Related to #85557 

## Proposed Changes

This is a very rough PoC, needs cleanup, code splitting and loading states on image generation.

* add roundtrip to get token
* add image generation request to API

## Testing Instructions

Open Calypso home on one of your sites, look for the quick link "Create logo with Jetpack AI". 
Enter a prompt and click generate. After a few seconds, you should see the LogoPresenter component show the `revised_prompt` and the generated image on the left.
